include/swift/SIL/SILDebugScope.h

```diff
@@ -64,6 +64,10 @@ class SILDebugScope : public SILAllocated<SILDebugScope> {
   /// into.
   SILFunction *getParentFunction() const;
 
+  PointerUnion<const SILDebugScope *, SILFunction *> getParent() const {
```

Comment:
I think we should either have an accessor, or make the field public, but not both.

Reply:
Done.